sdk/ml/azure-ai-ml/azure/ai/ml/entities/_job/automl/nlp/automl_nlp_job.py

```diff
@@ -123,6 +175,11 @@ def log_verbosity(self, value: Union[str, LogVerbosity]):
 
     @property
     def limits(self) -> NlpLimitSettings:
+        """Limit settings for NLP jobs
+
+        :return:
```

Comment:
Nit: Missing documentation for return value. May be Limit settings for NLP job or something